windu.el

```diff
@@ -362,10 +388,55 @@ After the split, both windows aim to have `windu-fill-column` width."
   (interactive)
   (select-window (windu-swap-buffers-in-direction 'below)))
 
+;;; window configuration end-user interactive wrapper functions
+(defun windu-set-window-configuration-2 ()
+  "Save the current window configuration to 'windu-window-configuration-2."
+  (interactive)
+  (windu-set-window-configuration 'windu-window-configuration-2))
+
+(defun windu-set-window-configuration-3 ()
+  "Save the current window configuration to 'windu-window-configuration-3."
+  (interactive)
+  (windu-set-window-configuration 'windu-window-configuration-3))
+
+(defun windu-set-window-configuration-4 ()
+  "Save the current window configuration to 'windu-window-configuration-4."
+  (interactive)
+  (windu-set-window-configuration 'windu-window-configuration-4))
+
+(defun windu-set-window-configuration-5 ()
+  "Save the current window configuration to 'windu-window-configuration-5."
+  (interactive)
+  (windu-set-window-configuration 'windu-window-configuration-5))
+
+(defun windu-load-window-configuration-2 ()
+  "Load the window configuration stored in 'windu-window-configuration-2, if one exists."
+  (interactive)
+  (windu-load-window-configuration windu-window-configuration-2)
+  (message "Window configuration loaded"))
```

Comment:
Any use in saying "Loaded window configuration 2" (or something) instead?  Like if you hit this sequence by accident, it might help you figure out which key you just mistakenly pressed?

Wish there was a way of summarizing the saved state ("4 windows open") or something, but I think this built-in API is too limited.